multicluster/hack/update-codegen-dockerized.sh

```diff
@@ -60,5 +59,8 @@ $GOPATH/bin/informer-gen \
   --output-package "${ANTREA_PKG}/multicluster/pkg/client/informers" \
   --go-header-file hack/boilerplate.go.txt
 
+go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1
```

Comment:
shouldn't this be added to the Docker image instead?

Reply:
Done